src/instance.ts

```diff
@@ -378,7 +377,7 @@ class Instance extends common.GrpcServiceObject {
    *
    * @param {GetBackupOperationsOptions} [options] The query object for listing
    *     backup operations.
-   * @param {CallOptions} [options.gaxOptions] The request configuration
+   * @param {grpc.CallOptions} [options.gaxOptions] The request configuration
```

Comment:
should be `CallOptions` imported above from `google-gax`

Reply:
Fixed everywhere